src/packages/solid/components/Badge/Badge.types.ts

```diff
@@ -0,0 +1,27 @@
+import type { NodeStyles, TextProps } from '@lightningjs/solid';
+import type { ComponentStyleConfig, NodeStyleSet, Tone, TextStyleSet } from 'types/types.js';
+import type { UIComponentProps } from 'types/interfaces.js';
+
+export interface BadgeProps extends UIComponentProps, BadgeStyleProperties {
+  /**
+   * Badge text
+   */
+  title?: TextProps['children'];
+}
+
+export type BadgeStyleProperties = Partial<{
```

Comment:
can we add comments around these properties similar to how we're adding them for standard props? since these are usable inline now, we should treat them the same as functional props